Plutarch/DataRepr/Internal/Field.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FunctionalDependencies #-}
```

Comment:
I *believe* this is redundant. I forgot to remove it during my cleanup.

Reply:
it was indeed redundant, thanks!